src/vnsw/agent/vrouter/ksync/ksync_flow_memory.cc

```diff
@@ -388,9 +405,9 @@ void KSyncFlowMemory::GetFlowTableSize() {
 }
 
 void KSyncFlowMemory::MapSharedMemory() {
-#if 0 //WINDOWSFIX
     GetFlowTableSize();
 
+#if !defined(_WIN32)
```

Comment:
I suggest you to use here ifndef like in the rest of the file

Reply:
Original file uses "#if defined" version (see lines: 6, 8).